hack/Makefile

```diff
@@ -17,7 +17,7 @@ PROW_TMP := $(shell mktemp -d)
 # and that's probably good enough for us.
 #
 # Please keep this in sync with
-# https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-sigs/wg-multi-tenancy/hnc-e2e.yaml.
+# https://github.com/kubernetes/test-infra/tree/master/config/jobs/kubernetes-sigs/hierarchical-namespaces/hnc-e2e.yaml.
```

Comment:
Can you please file an issue to set this up?

Reply:
https://github.com/kubernetes-sigs/hierarchical-namespaces/issues/5